### PR TITLE
Feat sp keystore

### PR DESCRIPTION
**Pull request by honeywest**
delete not used
rename var

### CODE REVIEW

File: primitives/keystore/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,6 @@ license = "Apache-2.0"
 homepage = "https://substrate.dev"
 repository = "https://github.com/paritytech/substrate/"
 description = "Keystore primitives."
-documentation = "https://docs.rs/sp-core"
```
Comment (thiolliere) [inline primitives/keystore/Cargo.toml:10]: why did you remove the doc ?

**Review by thiolliere — CHANGES_REQUESTED** [review]

File: primitives/keystore/Cargo.toml (hunk 1 above)
Comment (honeywest) [reply primitives/keystore/Cargo.toml:10]: It's a mistake, isn't it?

Comment (thiolliere) [reply primitives/keystore/Cargo.toml:10]: documentation is not a valid cargo keyword ?
I don't think it is a mistake, the link is good.

Comment (honeywest) [reply primitives/keystore/Cargo.toml:10]: ok, thank you

**Review by thiolliere — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by thiolliere** [comment]
bot merge
